src/lib/ai/mental-llama/MentalLLaMAAdapter.ts

```
@@ -202,9 +237,35 @@ export class MentalLLaMAAdapter {
             logger.warn(
               'CRISIS DETECTED BY TASK ROUTER. Initiating crisis protocol.',
-              { textSample: text.substring(0, 100) },
+              {
+                textSample: text.substring(0, 100),
+                userId: routeContext.userId,
+                sessionId: routeContext.sessionId,
+              },
             )
-            effectiveCategories = ['depression', 'anxiety', 'stress'] // Broad categories for crisis context
+            if (this.crisisNotifier) {
+              const alertContext: import('@/lib/services/notification/NotificationService').CrisisAlertContext =
```

ESLint

[error] `import()` type annotations are forbidden.

(@typescript-eslint/consistent-type-imports)